src/components/RadioButton/radio.module.scss

```diff
@@ -122,14 +129,156 @@
     }
 
     .selector-label {
-        font-size: $text-font-weight-regular;
+        font-size: $text-font-size-2;
 
         &-end {
-            margin-left: $space-xs;
+            margin-left: $space-xxs;
         }
 
         &-start {
-            margin-right: $space-xs;
+            margin-right: $space-xxs;
+        }
+    }
+
+    &-large {
+        input {
+            & + label {
+                .radio-button {
+                    height: $radio-large-height;
+                    width: $radio-large-width;
+
+                    &:after {
+                        top: $radio-large-after-top;
+                        left: $radio-large-after-left;
```

Comment:
might be able to save us from some of this repeated precision with more generic styling on the base case
top: 50%;
margin-top: calc(-50%);
left: ?
margin-left: ?

Reply:
That approach has cross-browser pitfalls.